tools/packaging/test/test_monkeyYaml.py

```diff
@@ -66,6 +66,14 @@ def test_Multiline_1(self):
         self.assertEqual(lines, [])
         self.assertEqual(value, yaml.load(y))
 
+    def test_Multiline_2(self):
```

Comment:
This should be 3 and the previous one should be 2 (right now there are 2 test_Multiline_1)